server/core/config/parser_validator_test.go

```diff
@@ -1100,38 +1095,20 @@ workflows:
 	Ok(t, err)
 
 	r := config.ParserValidator{}
-	globalCfgArgs := valid.GlobalCfgArgs{
-		AllowRepoCfg:  false,
-		MergeableReq:  false,
-		ApprovedReq:   false,
-		UnDivergedReq: false,
-	}
 
-	_, err = r.ParseRepoCfg(tmpDir, valid.NewGlobalCfgFromArgs(globalCfgArgs), "repo_id")
+	_, err = r.ParseRepoCfg(tmpDir, valid.NewGlobalCfg(), "repo_id")
```

Comment:
side note:

would be cool to simplify our config story, we have so many things that just get confusing. RepoConfig, GlobalConfig, UserConfig, lol.  Wonder how we can make this better.

Reply:
yeah, there are too many options to configure your repo.